kedro/framework/context/context.py

```diff
@@ -290,7 +291,7 @@ def _get_catalog(
             feed_dict=feed_dict,
             save_version=save_version,
```

Comment:
Can `save_version` ever be different from `session_id` now actually? 🤔  On seconds thoughts maybe we don't need to keep this in the hook spec after all if it's just a duplicate of `save_version`?

Reply:
At the moment they will always be the same, but this goes back to the question whether we should allow users to have a custom `save_version`, which requires user research. With the mindset of "don't add things that could potentially be used in the future", I'll remove it for now.